sources/packages/web/src/components/aest/InstitutionUserSummary.vue

```diff
@@ -1,131 +1,126 @@
-      <div>
-        <v-btn @click="searchUserTable()" color="primary" tile class="ml-2">
-          <font-awesome-icon icon="search" />
-        </v-btn>
-      </div>
-      <div v-if="clientType === ClientIdType.Institution" class="mx-2">
-        <v-btn class="float-right" @click="openNewUserModal()" color="primary">
-          <v-icon size="25" left> mdi-plus-circle </v-icon>
-          Add New User
-        </v-btn>
+  <!-- This component is shared between ministry and student users -->
+  <div class="row">
+    <div class="col align-self-start">
+      <span class="category-header-large color-blue">
+        All Users({{ usersListAndCount.totalUsers }})
+      </span>
+    </div>
+    <div>
+      <InputText
+        v-model="searchBox"
+        placeholder="Search User"
+        @keyup.enter="searchUserTable()"
+      />
+    </div>
+    <div>
+      <v-btn @click="searchUserTable()" color="primary" tile class="ml-2">
+        <font-awesome-icon icon="search" />
+      </v-btn>
+    </div>
+    <div v-if="clientType === ClientIdType.Institution" class="mx-2">
+      <v-btn class="float-right" @click="openNewUserModal()" color="primary">
+        <v-icon size="25" left> mdi-plus-circle </v-icon>
+        Add New User
+      </v-btn>
 
-        <!-- Add user -->
-        <AddInstitutionUser
-          :userType="userType"
-          :showAddUser="showAddUser"
-          @updateShowAddInstitutionModal="updateShowAddInstitutionModal"
-          @getAllInstitutionUsers="getAllInstitutionUsers"
-        />
+      <!-- Add user -->
+      <AddInstitutionUser
+        :userType="userType"
+        :showAddUser="showAddUser"
+        @updateShowAddInstitutionModal="updateShowAddInstitutionModal"
+        @getAllInstitutionUsers="getAllInstitutionUsers"
+      />
 
-        <!-- edit user -->
-        <EditInstitutionUser
-          :userType="userType"
-          :showEditUser="showEditUser"
-          :institutionUserName="institutionUserName"
-          @updateShowEditInstitutionModal="updateShowEditInstitutionModal"
-          @getAllInstitutionUsers="getAllInstitutionUsers"
-        />
-      </div>
+      <!-- edit user -->
+      <EditInstitutionUser
+        :userType="userType"
+        :showEditUser="showEditUser"
+        :institutionUserName="institutionUserName"
+        @updateShowEditInstitutionModal="updateShowEditInstitutionModal"
+        @getAllInstitutionUsers="getAllInstitutionUsers"
+      />
     </div>
-    <ContentGroup class="mt-2">
-      <DataTable
-        :value="usersListAndCount.users"
-        :lazy="true"
-        :paginator="true"
-        :rows="DEFAULT_PAGE_LIMIT"
-        :rowsPerPageOptions="PAGINATION_LIST"
-        :totalRecords="usersListAndCount.totalUsers"
-        @page="paginationAndSortEvent($event)"
-        @sort="paginationAndSortEvent($event)"
-        :loading="loading"
+  </div>
+  <ContentGroup class="mt-2">
+    <DataTable
+      :value="usersListAndCount.users"
+      :lazy="true"
```

Comment:
lazy and paginator are static values so can they be used like lazy ="true" ?

Reply:
![image](https://user-images.githubusercontent.com/77353155/146432025-31fcc48e-fd9a-429e-aff3-0f6589b24a94.png)
if we give `lazy ="true"` `true` is considered as string, `:lazy="true"` will be boolean

Reply:
Got it. 👍